config/resources/domain.json

```diff
@@ -185,6 +187,38 @@
       },
       "features": ["include-uri"],
       "new-resource-base": "http://timekeeper.redpencil.io/timesheets/"
+    },
+    "holiday-counters": {
```

Comment:
I would opt for a more generic name like `counters` or even `quantities`. Since the quantity-kind is a relation, it can serve multiple purposes in the future.